src/Stuart/Client.php

```diff
@@ -31,6 +33,24 @@ public function validateJob(Job $job)
         }
     }
 
+    /**
+     * @param $city
+     * @param $type pickup or dropoff
+     * @param \DateTime $dateTime
+     *
+     * @return SchedulingSlots|object
+     */
+    public function getSchedulingSlots($city, $type, \DateTime $dateTime)
```

Comment:
I would create two methods instead, and hard code `picking` and `delivering`:
`getSchedulingSlotsAtPickup` and `getSchedulingSlotsAtDropoff`